modal/_output.py

```diff
@@ -175,17 +175,14 @@ def __init__(
         self._app_page_url = None
         self._show_image_logs = False
 
+    def is_visible(self) -> bool:
+        return self._visible_progress
```

Comment:
Doesn't really matter, but the name suggests to me that this would be a property

```python
if self.is_visible:
    ...
```

(Although I guess it's just returning an attribute value anyway; why not just expose that and save the function call?)

Reply:
I almost made it a property, but I'm planning to change this to something slightly different in a second anyway – it will probably something more like `if OutputManager.is_enabled()`

since it will be on a class level, I don't think you can make it a property